ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/DC1-L2LEAF-TG-TEST-1A.cfg

```diff
@@ -0,0 +1,271 @@
+   trunk group UPLINK
+!
+vlan 112
+   name Tenant_A_OP_Zone_3
+   trunk group MLAG
+   trunk group UPLINK
+!
+vlan 120
+   name Tenant_A_WEB_Zone_1
+   trunk group MLAG
+   trunk group UPLINK
+!
+vlan 121
+   name Tenant_A_WEBZone_2
+   trunk group MLAG
+   trunk group UPLINK
+!
+vlan 130
+   name Tenant_A_APP_Zone_1
+   trunk group MLAG
+   trunk group UPLINK
+!
+vlan 131
+   name Tenant_A_APP_Zone_2
+   trunk group MLAG
+   trunk group UPLINK
+!
+vlan 132
+   name Tenant_A_APP_Zone_3
+   trunk group MLAG
+   trunk group UPLINK
+!
+vlan 140
+   name Tenant_A_DB_BZone_1
+   trunk group MLAG
+   trunk group UPLINK
+!
+vlan 141
+   name Tenant_A_DB_Zone_2
+   trunk group MLAG
+   trunk group UPLINK
+!
+vlan 150
+   name Tenant_A_WAN_Zone_1
+   trunk group MLAG
+   trunk group UPLINK
+!
+vlan 151
+   name svi_with_no_tags
+   trunk group MLAG
+   trunk group UPLINK
+!
+vlan 152
+   name svi_with_trunk_groups
+   trunk group MLAG
+   trunk group TENANT_A
+   trunk group TEST_TRUNK_GROUP1
+   trunk group UPLINK
+!
+vlan 160
+   name Tenant_A_VMOTION
+   trunk group MLAG
+   trunk group UPLINK
+!
+vlan 161
+   name Tenant_A_NFS
+   trunk group MLAG
+   trunk group UPLINK
+!
+vlan 162
+   name l2vlan_with_no_tags
+   trunk group MLAG
+   trunk group UPLINK
+!
+vlan 163
+   name l2vlan_with_trunk_groups
+   trunk group MLAG
+   trunk group TENANT_A
+   trunk group TEST_TRUNK_GROUP2
+   trunk group UPLINK
+!
+vlan 210
+   name Tenant_B_OP_Zone_1
+   trunk group MLAG
+   trunk group UPLINK
+!
+vlan 211
+   name Tenant_B_OP_Zone_2
+   trunk group MLAG
+   trunk group UPLINK
+!
+vlan 250
+   name Tenant_B_WAN_Zone_1
+   trunk group MLAG
+   trunk group UPLINK
+!
+vlan 310
+   name Tenant_C_OP_Zone_1
+   trunk group MLAG
+   trunk group UPLINK
+!
+vlan 311
+   name Tenant_C_OP_Zone_2
+   trunk group MLAG
+   trunk group UPLINK
+!
+vlan 350
+   name Tenant_C_WAN_Zone_1
+   trunk group MLAG
+   trunk group UPLINK
+!
+vlan 410
+   name Tenant_D_v6_OP_Zone_1
+   trunk group MLAG
+   trunk group UPLINK
+!
+vlan 411
+   name Tenant_D_v6_OP_Zone_2
+   trunk group MLAG
+   trunk group UPLINK
+!
+vlan 412
+   name Tenant_D_v6_OP_Zone_1
+   trunk group MLAG
+   trunk group UPLINK
+!
+vlan 450
+   name Tenant_D_v6_WAN_Zone_1
+   trunk group MLAG
+   trunk group UPLINK
+!
+vlan 451
+   name Tenant_D_v6_WAN_Zone_2
+   trunk group MLAG
+   trunk group UPLINK
+!
+vlan 452
+   name Tenant_D_v6_WAN_Zone_3
+   trunk group MLAG
+   trunk group UPLINK
+!
+vlan 4091
+   name MLAG_PEER
+   trunk group MLAG
+!
+vrf instance MGMT
+!
+interface Port-Channel1
+   description DC1_LEAF_TG_TEST_1_Po16
+   no shutdown
+   switchport
+   switchport mode trunk
+   switchport trunk group UPLINK
+   mlag 1
+!
+interface Port-Channel3
+   description MLAG_PEER_DC1-L2LEAF-TG-TEST-1B_Po3
+   no shutdown
+   switchport
+   switchport trunk allowed vlan 2-4094
+   switchport mode trunk
+   switchport trunk group MLAG
+!
+interface Port-Channel23
+   description server23_trunk_group_tests_
```

Comment:
Looks like something is missing here?

Reply:
I did not touch this code, but indeed it looks like our description template is requiring some variable I am not setting.